definitions/infra-awsrekognition/summary_metrics.yml

```diff
@@ -0,0 +1,26 @@
+providerAccountName:
+  title: AWS account
+  unit: STRING
+  tag:
+    key: providerAccountName
+region:
+  title: Region
+  unit: STRING
+  tag:
+    key: aws.awsRegion
+requests:
+  goldenMetric: requests
+  title: Total Request Count
+  unit: COUNT
+userError:
+  goldenMetric: userError
+  title: User Error count
+  unit: COUNT
+runtimeUserErrors:
+  goldenMetric: runtimeUserErrors
+  title: RuntimeUserErrors count
```

Comment:
For consistency, the title should be "Runtime User Errors count"

Reply:
addressed